packages/opentelemetry-types/src/metrics/Metric.ts

```diff
@@ -54,6 +54,12 @@ export interface MetricOptions {
    */
   monotonic?: boolean;
 
+  /**
+   * (Measure only) Absolute asserts that this metric cannot only accept positive values.
```

Comment:
It looks like this the same as `opentelemetry-metrics/src/types.ts` but it has the opposite meaning, is this correct ?
1. (Measure only) Absolute asserts that this metric `will` only accept positive values. 
2. (Measure only) Absolute asserts that this metric `cannot` only accept positive values.

Reply:
```suggestion
   * (Measure only) Asserts that this metric may only have a positive value.
```